Makefile

```diff
@@ -1,10 +1,10 @@
-AWS_ACCOUNT_NAME ?= michael
-AWS_DEFAULT_REGION ?= eu-west-1
+AWS_DEFAULT_REGION ?= eu-east-1
```

Comment:
```suggestion
AWS_DEFAULT_REGION ?= us-east-1
```

Reply:
It's funny, because I changed it to `east` 🤦‍♂